ONSdigital/florence
# Fix/viewer see completed collection

### What

* Viewers should see completed unpublished collections
* Tests added for above 

### How to review

1. Create a collection
2. Add contents to the collection, 
3. Review all contents of the collection
4. Approve collection 
5. Note this should move to publish queue
6. Note this should no longer be visible on the select collection screen
7. Change user to a viewer (make sure this viewer is in a team that has access to the collection
8. Note the collection should be visible on the select collection screen

### Who can review

Anyone except me
